### PR TITLE
chore(deps): bump sdk

### PR DESCRIPTION
**Pull request by alia-bitgo**
TICKET: WP-598

### CODE REVIEW

**Review by mohammadalfaiyazbitgo — APPROVED** [review]

File: src/containers/CreateBroadcastableTransaction/CreateBroadcastableTransactionIndex.tsx [hunk 1]
```diff
@@ -18,9 +18,11 @@ function isBroadcastableTransaction(
...
   return (
     broadcastableTransaction &&
-    broadcastableTransaction.length !== undefined &&
-    broadcastableTransaction.length > 0 &&
-    broadcastableTransaction[0].serializedTx !== undefined
+    broadcastableTransaction.transactions &&
+    broadcastableTransaction.transactions.length !== undefined &&
+    broadcastableTransaction.transactions.length > 0 &&
+    broadcastableTransaction.transactions[0].serializedTx !== undefined &&
+    broadcastableTransaction.lastScanIndex !== undefined
```
Comment (zahin-mohammad) [inline src/containers/CreateBroadcastableTransaction/CreateBroadcastableTransactionIndex.tsx:25]: Nit: This should be a different commit

**Review by zahin-mohammad — APPROVED** [review]
